### PR TITLE
Use information from config in setup.py

### PR DESCRIPTION
**Pull request by pulkitvyas08**
Completing #11
Please tell me what more do I need to add in the config.py file

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -19,10 +19,10 @@
...
 
 setup(
-    name='facerec',
+    name=config.name,
     version=config.__version__,
-    license='BSD',
-    url='http://github.com/rushabh-v/linux_face_unlock',
+    license=config.license,
+    url=config.url,
     description="A face Authentication system for Ubuntu Linux.",
```
Comment (rushabh-v) [inline setup.py]: Just Capitalise all the variable names (except `__version__`) in the config.py and here as well. And rename `url` to `GIT_URL`.

**Review by rushabh-v — CHANGES_REQUESTED** [review]

**Comment by rushabh-v** [comment]
Resolves #11

**Comment by rushabh-v** [comment]
And remove the `version` variable from config as it's unused.

**Comment by rushabh-v** [comment]
give @pulkitvyas08 5 points

**Comment by rushabh-v** [comment]
Let it be the list of those many variables only for now. We will add more in the future if we find anything worthy to be there!

**Comment by pulkitvyas08** [comment]
noted

**Review by rushabh-v — APPROVED** [review]

**Pull request merged**